examples/website/geospatial/.babelrc

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Do we really need this file. Other examples on loaders.gl don't have such file.

Reply:
my feeling - yes. Because i've got the following error 
![image](https://user-images.githubusercontent.com/87765428/129536675-eb1e0093-3815-438e-8735-08fdbdd84c78.png)
Also, have found several issues on stackoverflow and github, e.g. https://github.com/babel/babel/issues/9849
Probably, it is connected with async/await

Reply:
This should definitely not be needed. I don't think any other example has a custom .babelrc. Perhaps some module is not including the `@babel/runtime` dependency.